### PR TITLE
[485] Update financial incentives list

### PR DESCRIPTION
**Pull request by darokel**
### Context

https://trello.com/c/Jgi2ZgGI/485-financial-incentives-new-figures

### Changes proposed in this pull request

- Update the financial incentive figures for the next cycle

### Guidance to review

Check the figures with the trello ticket. This PR doesn't actually seed in the values. We won't be able to do that until we end rollover and begin the new cycle. A follow up PR will occur

### Checklist

- [x] Make sure all information from the Trello card is in here
- [x] Attach to Trello card
- [x] Rebased main
- [x] Cleaned commit history
- [x] Tested by running locally

### CODE REVIEW

**Comment by darokel** [comment]
[Financial incentives - new figures](https://trello.com/c/Jgi2ZgGI/485-financial-incentives-new-figures)

File: app/services/subjects/financial_incentive_creator_service.rb [hunk 1]
```diff
@@ -8,6 +8,25 @@ def initialize(year:, subject: Subject, financial_incentive: FinancialIncentive)
 
     def subject_and_financial_incentives
       subject_and_financial_incentives = {
+        2023 => {
+          %w[Mathematics Physics Chemistry Computing] => {
+            bursary_amount: "27000",
+            scholarship: "29000",
+          },
```
Comment (defong) [inline app/services/subjects/financial_incentive_creator_service.rb:15]: How about these?

"Latin", "Ancient Greek", "Ancient Hebrew"

File: app/services/subjects/financial_incentive_creator_service.rb [hunk 2]
```diff
@@ -8,6 +8,25 @@ def initialize(year:, subject: Subject, financial_incentive: FinancialIncentive)
...
+          ["French", "German", "Italian", "Japanese", "Mandarin", "Modern languages (other)", "Russian", "Spanish"] => {
+            bursary_amount: "25000",
+            scholarship: "27000",
```
Comment (defong) [inline app/services/subjects/financial_incentive_creator_service.rb:18]: How about these?

"Latin", "Ancient Greek", "Ancient Hebrew"

Comment (darokel) [reply app/services/subjects/financial_incentive_creator_service.rb:18]: If they count as part of that set I'll add them in!

Comment (darokel) [reply app/services/subjects/financial_incentive_creator_service.rb]: done

**Review by goodviber — APPROVED** [review]

**Pull request merged**